gateway/gateway.ts

```diff
@@ -49,16 +49,24 @@ class ServiceRemoteGraphQLDataSource extends RemoteGraphQLDataSource {
   }
 }
 
-export const gateway = new ApolloGateway({
-  serviceList: getServices(),
-  experimental_pollInterval: IS_DEV ? 5000 : 0,
-  buildService({ name, url }) {
-    if (name === "logout") {
-      return new LocalGraphQLDataSource(logoutSchema);
-    }
+export const createGateway = () => {
+  const options: any = {};
 
-    return new ServiceRemoteGraphQLDataSource({
-      url,
-    });
-  },
-});
+  if (IS_DEV) {
+    options.serviceList = getServices();
```

Comment:
on dev for now we still have a local list of services

maybe in the future we could use apollo studio development schemas, but it seems overkill